### PR TITLE
test(e2e): cloudfront use tls termination

### PR DESCRIPTION
**Pull request by iamhopaul123**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Review by Lou1415926 — APPROVED** [review]

File: e2e/cloudfront/copilot/environments/test/manifest.yml [hunk 1]
```diff
@@ -10,4 +10,5 @@ http:
         restrict_to:
           cdn: true
 
-cdn: true
\ No newline at end of file
+cdn:
+  terminate_tls: true
```
Comment (dannyrandall) [inline e2e/cloudfront/copilot/environments/test/manifest.yml:14]: i think this will fail validation without a cert...

Comment (iamhopaul123) [reply e2e/cloudfront/copilot/environments/test/manifest.yml:14]: let me test...

Comment (iamhopaul123) [reply e2e/cloudfront/copilot/environments/test/manifest.yml:14]: ok this is not expected at all. I'll work on a fix before merging this PR.

**Review by dannyrandall — APPROVED** [review]
should be good once #4096 is merged! thanks for fixing that!

**Pull request merged**